package.json

```diff
@@ -7,7 +7,7 @@
     "test": "forge test -v",
     "prettier:check": "prettier --check \"(contracts|test|script)/**/*.sol\"",
     "prettier:fix": "prettier --write \"(contracts|test|script)/**/*.sol\"",
-    "lint": "solhint -w 3 'contracts/**/*.sol' 'test/**/*.sol' 'script/**/*.sol'"
+    "lint": "solhint 'contracts/**/*.sol' 'test/**/*.sol' 'script/**/*.sol'"
```

Comment:
-w 3 says that it should allow maximum 3 warnings, should we disable that?

Reply:
Yes, there are 1146 warnings in total, should we fix them all?

Reply:
I get your point, yes we should fix them all but not in this pr.